### ISSUE TITLE
Results is 

### ISSUE THREAD

**Issue by chjXu** [GengDavid/pytorch-cpn#38]
I used the weight file of 10 epochs of training to test the image. Why is the accuracy so low? Did I train too little?
![2020-10-15 16-23-46屏幕截图](https://user-images.githubusercontent.com/52600391/96097929-216cdb00-0f04-11eb-9e04-4d52dacad259.png)
